components/control_bar.py

```diff
@@ -77,11 +59,21 @@ def layout():
                         "Image transformations",
                         "fluent-mdl2:image-pixel",
                         "image-transformations",
-                        children=[
-                            _color_selector_control("red"),
-                            _color_selector_control("blue"),
-                            _color_selector_control("green"),
-                        ],
+                        children=html.Div(
+                            [
+                                dmc.Text("Colormap scalar range", size="sm"),
+                                dmc.RangeSlider(
+                                    id=f"colormap-scale",
+                                    value=[0, 255],
```

Comment:
Could you set the defaults to be a range that enhances the contrast? On initial load it's hard to tell that there's actually an image.